save-cloud-charts/save-cloud/values.yaml

```diff
@@ -23,7 +23,7 @@ backend:
 backend_cosv:
```

Comment:
should it be named to cosv_backend?

Reply:
I think this can not be changed. Our frontend is named according to the same logic.
![Снимок экрана 2024-02-20 в 14 53 01](https://github.com/saveourtool/save-cloud/assets/50615459/5adad535-92f2-48dc-9f9c-533fe034c30f)

Reply:
so may be rename frontend too?